assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -18,4 +19,39 @@ export const tests: Tests = {
     independent: true,
     seed: 3,
   },
+  newPaymentFlow: {
+    variants: ['control'],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    // TODO: turn to false
+    isActive: true,
+    independent: true,
+    seed: 4,
+  },
+};
+
+export const newContributionsFlowTests: Tests = {
+  newPaymentFlow: {
+    variants: ['newPaymentFlow'],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    // TODO: turn to false
+    isActive: true,
+    independent: true,
+    seed: 5,
+  },
+};
+
+export const allTests: Tests = {
```

Comment:
whoops, I've just realised this is a bad idea - people landing on any other page on the site would always get put in the `newPaymentFlow` variant (since it overwrites the first) and then be stuck in that even if they were in the control via the serverside test